bin/api-docs/gen-components-docs/markdown/index.js

```diff
@@ -0,0 +1,44 @@
+/**
+ * External dependencies
+ */
+const json2md = require( 'json2md' );
+
+/**
+ * Internal dependencies
+ */
+const { generateMarkdownPropsJson } = require( './props' );
+
+function generateMarkdownDocs( { typeDocs, subcomponentTypeDocs } ) {
+	const mainDocsJson = [
+		'<!-- This file is generated automatically and cannot be edited directly. -->\n',
+		{ h1: typeDocs.displayName },
+		{
+			p: `<p class="callout callout-info">See the <a href="https://wordpress.github.io/gutenberg/?path=/docs/components-${ typeDocs.displayName.toLowerCase() }--docs">WordPress Storybook</a> for more detailed, interactive documentation.</p>`,
```

Comment:
Should we have similar callouts for private, deprecated, and experimental components?

Reply:
I will add one for experimental when we encounter it. But probably not for private, since private component readmes shouldn't be public on Block Editor Handbook.

Reply:
Makes sense. And what about deprecated? Same strategy as experimental?

Reply:
Right. Probably a customizable callout, for example:

https://github.com/WordPress/gutenberg/blob/ca9857ec13f97ff75fdc83dba3cb00e5630d45b7/packages/components/src/radio-group/README.md?plain=1#L3-L5

Reply:
Makes sense. Do you think it would make sense to include an example of an experimental component and of a deprecated component in this PR or as a quick follow-up once this PR is merged?

Reply:
Yes! After merging this first iteration I'm going to continue adding at least around ten more components so I can test and add more features as necessary. Once it feels somewhat stable we can open it up to other contributors.